example/package.json

```diff
@@ -3,7 +3,7 @@
   "scripts": {
     "start": "node dist/server.js",
     "prestart": "npm run build",
-    "build": ". build.sh"
+    "build": "./build.sh"
```

Comment:
I think this should be:
  . ./build.sh

I didn't know what `.` does and had to look up; for a better clarity, I propose:
  source ./build.sh